opacus/accountants/utils.py

```diff
@@ -13,26 +13,20 @@
 # limitations under the License.
 
 from opacus.accountants import create_accountant
-
-
-# min range bound when searching for sigma given epsilon
-DEFAULT_SIGMA_MIN_BOUND = 0.01
-# starting point for a max range bound when searching for sigma given epsilon
-DEFAULT_SIGMA_MAX_BOUND = 10
-# condition to halt binary search for sigma given epsilon
-SIGMA_PRECISION = 0.01
-# max possible value for returned sigma.
-# Noise higher than MAX_SIGMA considered unreasonable
-MAX_SIGMA = 2000
+from typing import Optional
 
 
 def get_noise_multiplier(
     *,
     target_epsilon: float,
     target_delta: float,
     sample_rate: float,
-    epochs: int,
+    epochs: Optional[int] = None,
+    steps: Optional[int] = None,
     accountant: str = "rdp",
+    epsilon_tolerance: float = 0.01,
+    sigma_low: float = 0.01,
```

Comment:
I wanted to ask, why the default `sigma_low` is so high. We will surprise the user with the warning and suboptimal result if the best sigma is 1e-3 for example.
But then I found that I don't understand what is the point of having this parameter at all? Why not starting with sigma_low=0 always? 

This won't hurt this expression `target_epsilon - eps_high > epsilon_tolerance`. We will never reach this value closer than tolerance/2.